FONTAWESOME5.md

```diff
@@ -0,0 +1,123 @@
+# FontAwesome 5
+
+FontAwesome 5 is split into three different font files which makes it somewhat
+harder to use in some cases. The implemented solution should be fairly
+straightforward to use after it has been setup. 
+Android and iOS handles fonts differently which is why it could be
+easily implemented without any additional setup.
+
+### Table of Content
+
+* [`Setup`](#setup-ios-only)
+* [`Usage`](#usage)
+* [`Upgrading to Pro`](#upgrading-to-pro)
+
+# Setup (iOS only)
+
+**If you intend to use FontAwesome5 Pro you should upgrade before linking the
+project, it's just easier that way**
+
+Using ```react-native link react-native-vector-icons``` should do most of the
+work needed but if there is any general problem linking the library, look in
+the [`main readme`](README.md) for a solution.
+
+After the library has been successfully linked to your project you need to
+setup the fonts in the beginning of your AppDelegate.m.
+
+```objc
+#import "AppDelegate.h"
+
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTRootView.h>
+#import <RNVectorIconsManager.h>            // <----- ADD THIS LINE
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [RNVectorIconsManager setupFontAwesome5]; // <----- ADD THIS LINE
```

Comment:
Having to do this manual integration makes me a bit sad. I'm sure you had to, but would be awesome if it was possible to avoid it. What was the cause of having to resort to this?

Reply:
The way iOS looks up fonts using font families is frustrating since we need to separate them in order to use a specific style. However, based on the information iOS extracts, they are the same and Regular would always be used. So setupFontAwesome5 adds some extra traits that can be defined from javascript and selecting the style the user wants.